Project.toml

```diff
@@ -20,11 +20,11 @@ Aqua = "0.8"
 ChainRulesCore = "1"
 ChainRulesTestUtils = "1"
 FiniteDifferences = "0.12"
-LinearAlgebra = "<0.0.1, 1"
-Random = "<0.0.1, 1"
-Test = "<0.0.1, 1"
+LinearAlgebra = "1"
+Random = "1"
+Test = "1"
```

Comment:
Are you sure about these changes? A while ago I was told that "<0.0.1, 1" would be needed in some scenarios even on somewhat recent Julia versions. But I don't recall the details.

Reply:
Maybe related: [Adding compat entry to stdlib breaks tests on Julia 1.3 - General Usage - Julia Programming Language](https://discourse.julialang.org/t/adding-compat-entry-to-stdlib-breaks-tests-on-julia-1-3/105705)

Reply:
I think it was https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958#update-november-9th-2023-2 that I had in mind.

Reply:
The issue linked in the latest suggestion (of adding `<0.0.1`) seems to be present in Julia > 1.4 as well: https://discourse.julialang.org/t/why-random-jl-is-fixed-to-version-0-0-0/105957/2

Reply:
But do we really want to support this setup when we aren't testing it?

Reply:
Al so in that discourse someone said:

> Right, and the issue is only seen on older versions of Juli

Reply:
That's only correct if you do not perform any Pkg operations in your tests.

Reply:
But again, we haven't tested such a configuration and so it doesn't make sense to me why we would want to support it